### PR TITLE
On create PR panel, show the diffs between the two branch points you have selected

### PR DESCRIPTION
**Pull request by ppezaris**
**This PR Addresses:**  
[on create PR panel, show the diffs between the two branch points you have selected](https://trello.com/c/fuWMPLpM/4465-on-create-pr-panel-show-the-diffs-between-the-two-branch-points-you-have-selected)

### CODE REVIEW

**Comment by ppezaris** [comment]
looks good

**Comment by ppezaris** [comment]
test fully sync

**Comment by ppezaris** [comment]
new comment

**Comment by ppezaris** [comment]
this is going to github

File: shared/agent/src/protocol/agent.protocol.scm.ts [hunk 1]
```diff
@@ -375,3 +375,24 @@ export const FetchForkPointRequestType = new RequestType<
 	void,
 	void
 >("codestream/scm/forkPoint");
+
+export interface DiffBranchesRequest {
+	repoId: string;
+	baseRef: string;
```
Comment (ppezaris) [inline shared/agent/src/protocol/agent.protocol.scm.ts:381]: test comment

File: shared/agent/src/managers/scmManager.ts [hunk 2]
```diff
@@ -9,7 +9,10 @@ import {
 	CodeStreamDiffUriData,
 	FetchForkPointRequestType,
 	FetchForkPointRequest,
-	FetchForkPointResponse
+	FetchForkPointResponse,
```
Comment (ppezaris) [inline shared/agent/src/managers/scmManager.ts:12]: test 2

**Comment by bcanzanella** [comment]
test

File: shared/agent/src/git/gitService.ts [hunk 3]
```diff
@@ -378,6 +378,26 @@ export class GitService implements IGitService, Disposable {
 		return patches;
 	}
 
+	async diffBranches(
```
Comment (bcanzanella) [inline shared/agent/src/git/gitService.ts:381]: test

**Comment by bcanzanella** [comment]
test

**Comment by bcanzanella** [comment]
test2

**Comment by bcanzanella** [comment]
test

**Comment by bcanzanella** [comment]
test

File: shared/agent/src/git/gitService.ts [hunk 4]
```diff
@@ -378,6 +378,26 @@ export class GitService implements IGitService, Disposable {
...
+		repoPath: string,
+		baseRef: string,
+		headRef: string
```
Comment (bcanzanella) [inline shared/agent/src/git/gitService.ts:384]: test

Comment (bcanzanella) [inline shared/agent/src/git/gitService.ts:384]: test

Comment (bcanzanella) [inline shared/agent/src/git/gitService.ts:384]: asdasd

Comment (bcanzanella) [inline shared/agent/src/git/gitService.ts:384]: test

**Pull request closed without merging**